modular_skyrat/modules/loadouts/loadout_items/loadout_datum_masks.dm

```diff
@@ -76,6 +76,10 @@ GLOBAL_LIST_INIT(loadout_masks, generate_loadout_items(/datum/loadout_item/mask)
 	name = "Glass Gas Mask"
 	item_path = /obj/item/clothing/mask/gas/glass
 
+/datum/loadout_item/mask/gas_full
+	name = "Full Gas Mask"
+	item_path = /obj/item/clothing/mask/gas/full
+
```

Comment:
Not sure how I feel with people spawning in with a full gas mask, not gonna lie. I know it's for the whole vibe and everything, but I think that's perhaps a bit too far for the loadout, I'd rather keep it as a cargo goodie/maint loot instead.

Reply:
Will remove it then! Although it is strange to me that there's other gas masks in the loadout (and don't appear to be job-locked as well).